### ISSUE TITLE
how to generate (almost) unique question sheets ?

### ISSUE THREAD

**Issue by jboulen** [academic-moodle-cooperation/moodle-mod_offlinequiz#120]
In our university, we would like generate unique question sheets to our students, where each students will have, as much as possible, a different questions order.

With the current plugin version, can you confirm that we are only able to generate 6 question sheets (one question sheet for group A, B, C, D, E and F) ?

If it's not possible yet, would you interest by this feature ? How our university could help you to achieve this ?

**Comment by univietw** [comment]
I can confirm, you can have at most 6 groups at the moment and we have nothing planned at the moment to change that.
If you want to contribute you can send in a pull request. The only question for me is how to solve this huge amount of groups. You'd have to change the answer sheet which means a lot of work.
With the new algorithm it would be easier, but it is far from polished at the moment.

Long things short: I don't see a huge benefit in comparison to the work you'd have to do. But if you send in a pull request I'm will take a look at it for sure.

If you have any questions regarding the code feel free to ask.

**Comment by jboulen** [comment]
Sorry, I forgot to reply.
Initially, we had imagined to use binary notation with checkboxes for groups (last box checked means group "1", two last boxes checked means group "3", etc).
Following your answer, we decided in January that it will be better for us to wait your new algorithm.
Also, from our first feedbacks (we deployed your plugin last year), teachers seem to be satisfied with the current behavior. Maybe we had anticipated too much on a non-essential feature.
Thanks for your useful plugin.
